include/aspect/material_model/visco_plastic.h

```diff
@@ -304,6 +305,7 @@ namespace aspect
         {
           diffusion,
           dislocation,
+          frankkamenetskii,
```

Comment:
We use CamelCase to distinguish words in class names and type names, but underscores to do it in variable names:

```suggestion
          frank_kamenetskii,
```